### DISCUSSION TITLE
DSD files

### DISCUSSION THREAD

**Discussion by hrivica** [rern/rAudio#721] [Bug Reports]
First of all, thanks for all of the hard work.
I have Pi 3 and HiFiBerry DAC +.
There is no sound when I play DSD files.

**Comment by klassikmann** [comment]
@hrivica 
Is your devive capable to play native DSD files? Or via DoP (DSD over PCM) only? Or not at all?
Did you read the help at Player-Menu -> DSD over PCM?
> For DSD-capable devices without drivers dedicated for native DSD.
>  • Enable if there's static/no sound from the DAC which means not support as native DSD.
>  • DoP will repack 16bit DSD stream into 24bit PCM frames and transmit to the DAC. 
>  • PCM frames will be reassembled back to original DSD stream, COMPLETELY UNCHANGED, with expense of double bandwith.
>  • On-board audio and non-DSD devices will always get DSD converted to PCM stream, no bit-perfect

EDIT:
Additional information regarding MPD configuration (from [https://mpd.readthedocs.io/en/latest/user.html#direct-stream-digital-dsd](url)):

> DSD (Direct Stream Digital) is a digital format that stores audio as a sequence of single-bit values at a very high sampling rate. It is the sample format used on Super Audio CDs.
> MPD understands the file formats DSDIFF and DSF. There are three ways to play back DSD:
>   •    Native DSD playback. Requires ALSA 1.0.27.1 or later, a sound driver/chip that supports DSD and of course a DAC that supports DSD.
>   •    DoP (DSD over PCM) playback. This wraps DSD inside fake 24 bit PCM according to the DoP standard. Requires a DAC that supports DSD. No support from ALSA and the sound chip required (except for bit-perfect 24 bit PCM support).
>   •   Convert DSD to PCM on-the-fly.
> Native DSD playback is used automatically if available. DoP is only used if enabled explicitly using the dop option, because there is no way for MPD to find out whether the DAC supports it. DSD to PCM conversion is the fallback if DSD cannot be used directly.
